tests/scripts/custodian/untag-to-delete.yaml

```diff
@@ -0,0 +1,33 @@
+policies:
+# should be ran on CRON to remove on every saturday morning / friday night
+# note: userkeys here is an override var.
```

Comment:
Ok this is overriding the donotdelete tag. Once this is completed on late Friday early Saturday. After that what's the next cleanup? late Tuesday? considering the current custodian 2 day and assuming the first resource was created early Monday.

Reply:
currently we technically don't have a set cron, but my current idea is to:
* hourly - mark resources for deletion in 2 days from their creation time
* weekly on friday night - remove resources marked not to delete for the week only (normal / long living doNotDelete resources will stay alive)

so right now the answer is, it will still be checking for resources via the hourly job, and removing them if they are over 2 days old. I can add any of the following to amend this:
* add a job / modify this one to remove the resource directly if it has that tag instead of removing the tag, since this job will run 1x per week on friday nights -- this needs to be done before the 'release' of this, but I think it could be okay to leave as-is since these all are dry-run only at the moment. 
* modify the cron for the hourly job not to run on the weekends. This would not make much sense to me though but we could add it

Reply:
updated this job to remove resources tagged for friday removal instead of removing the tags.